xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -735,7 +735,8 @@ void CGUIDialogPVRTimerSettings::InitializeTypesList()
     return;
   }
 
-  int idx = 0;
+  bool foundThisType(false);
```

Comment:
I would prefer if you use a type prfix for the variable name => bFoundThisType
We do not do this in a consistent way in our code base, but to follow our coding guidelines we should obey the guideline for new code.